cmd/run/run_test.go

```diff
@@ -573,6 +573,80 @@ func TestBuildServerWithOIDCAuthentication(t *testing.T) {
 	}
 }
 
+func TestBuildServerWithOIDCAuthenticationAlias(t *testing.T) {
```

Comment:
Once https://github.com/openfga/openfga/pull/1417 is merged, please add unit tests as well.

Reply:
It's been merged

Reply:
@miparnisari if I add a test case to this test suite, would the one I added in `run_test` still be relevant? 
I have the feeling they both would test the same thing. Should I keep the two or only one in `oidc_test` ?

Reply:
It's OK to have both, they test the same feature but at different levels. `TestBuildServerWithOIDCAuthenticationAlias` is testing that the server setting `AuthnOIDCConfig` is being passed correctly downstream

Reply:
Done 😃